routes/git-hook.js

```diff
@@ -21,7 +22,7 @@ router.post('/', async (req, res) => {
 	if (branch !== 'dev' && branch !== 'main') {
 		return res.send('Automatic webhook updates are only enabled on dev and main branch');
 	}
-	await Tools.updateCode();
+	console.log(await Tools.updateCode());
```

Comment:
this log ok to keep in prod?

Reply:
> this log ok to keep in prod?

For now, ye. Until we find out why git-hook sux